calculate.ts

```diff
@@ -0,0 +1,154 @@
+import axios from "axios";
+import moment from "moment";
+import prompt from "prompt";
+import { calcGasLastNTrans } from "./transactions";
+import { getBlockNumber } from "./util";
+import logger from "./winston";
+require("dotenv").config();
+
+const { API_KEY, DATE_FORMAT, TIME_FORMAT, BSC_ADDR } = process.env;
```

Comment:
Should probably set defaults for TIME_FORMAT and BSC_ADDR, either use `moment.HTML5_FMT` values or just manually specify them

```suggestion
const { API_KEY, DATE_FORMAT = moment.HTML5_FMT.TIME_SECONDS, TIME_FORMAT = HH:mm:ss, BSC_ADDR } = process.env;
```